.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Report a problem for the OPA community to investigate
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Thanks for opening an issue to request a feature or file a bug!
+If you provide some basic information, it helps us address problems faster. -->
+
+## Short description 
+<!--
+Any information you think might be helpful. Examples include the environment
+where OPA was running (e.g., if inside Kubernetes, what resource limits did you configure
+OPA with?), how long OPA had been running for, what was happening around the time
+when you identified the problem, etc.
+-->
+
+## Steps To Reproduce
+<!--
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See an error
+-->
+
+## Expected behavior
+<!--
+Describe what you expected to happen.
+-->
+
+## Screenshots
+<!--
+If applicable, add screenshots to help explain your problem.
+-->
+
```

Comment:
Not sure how others feel but I'd rather not see screenshots in issues unless they're used to show visual/presentation issues in the CLI output, website, etc. In other words, I don't want to see screenshots of error messages or logs or `top` output, etc.